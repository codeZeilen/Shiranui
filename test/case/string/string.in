let unit = print("Hello");
let unit = print("Hello" + "World");
let unit = print("Hello" = "Hello");
let unit = print("Hello" = "World");
let unit = print("Hello" /= "Hello");
let unit = print("Hello" /= "World");
