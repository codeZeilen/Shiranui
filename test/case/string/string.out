Hello
HelloWorld
true
false
false
true
